der/src/error.rs

```diff
@@ -220,6 +220,13 @@ pub enum ErrorKind {
     #[cfg_attr(docsrs, doc(cfg(feature = "std")))]
     PermissionDenied,
 
+    /// Real related error: base is not DER compliant (base encoded in enum)
+    RealBaseInvalid(u8),
+    /// Real related error: encoded exponent cannot be represented on an IEEE-754 double
+    RealExponentTooLong,
+    /// Real related error: encoding not supported or malformed
+    RealISO6093Error,
+
```

Comment:
Not sure if I mentioned this before, but this is *a lot* of `REAL`-related error variants to add to what is primarily intended to be a cryptography library.

Pretty much every other type propagates errors like this as `Self::TAG.value_error()`, which only includes information about the tag.